libexec/rtld-elf/aarch64/rtld_c18n_asm.S

```diff
@@ -172,35 +192,8 @@ tramp_header_hook_def:
 	.chericap	0
 tramp_header_hook_function:
 	.chericap	0
```

Comment:
At this point, if neither of them has any code, just use a C struct for these things. The assembly doesn't reference the names, only the C does, so you gain nothing by having it in assembly.